lib/cisco_node_utils/interface_ospf.rb

```diff
@@ -250,6 +254,21 @@ def default_network_type
       config_get_default('interface_ospf', lookup)
     end
 
+    def mtu_ignore
+      config_get('interface_ospf', 'mtu_ignore', @interface.name)
+    end
+
+    # interface %s
```

Comment:
They are not necessary. But the set methods in this ruby file had these comments like this before and I added for the new ones as well for consistency. I can get rid of them if you want but if we do that, let us do for the entire file (unless there is a special need).